src/cascadia/TerminalSettingsEditor/InteractionViewModel.idl

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import "EnumEntry.idl";
+
+namespace Microsoft.Terminal.Settings.Editor
+{
+    runtimeclass InteractionViewModel : Windows.UI.Xaml.Data.INotifyPropertyChanged
+    {
+        InteractionViewModel(Microsoft.Terminal.Settings.Model.GlobalAppSettings globalSettings);
+        Microsoft.Terminal.Settings.Model.GlobalAppSettings GlobalSettings;

```

Comment:
```suggestion
        Microsoft.Terminal.Settings.Model.GlobalAppSettings GlobalSettings{ get; };
```
Do we need to expose the setter for this?

Reply:
Whoops, thanks for the catch!